pepper-apis/dsm-core/src/main/resources/liquibase/PEPPER-900_field_settings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="cunningh" id="PEPPER-900_field_settings">
+        <insert tableName="field_settings">
+            <column name="ddp_instance_id"
+                    valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='cmi-lms')"></column>
+            <column name="field_type" value="VOCABULARY"/>
+            <column name="column_name" value="REQUEST_STATUS"/>
+            <column name="display_type" value="OPTIONS"/>
+            <column name="possible_values"
+                    value='[{"value": "Needs Review"},{"value": "Don&apos;t Request"},{"value": "On Hold"},{"value": "Request"},{"value": "Sent"},{"value": "Received"},{"value": "Returned"},{"value": "Unable to Obtain"}]'/>
```

Comment:
Is that the github UI messing with the apostrophe should this be an escaped apostrophe: 
'[{"value": "Needs Review"},{"value": "Don\'t Request"},{"value": "On Hold"},{"value": "Request"},{"value": "Sent"},{"value": "Received"},{"value": "Returned"},{"value": "Unable to Obtain"}]'

Reply:
Oh for the love of....
It removes my escape and renders in this UI.

Reply:
<img width="833" alt="Screenshot 2023-06-16 at 10 55 58 AM" src="https://github.com/broadinstitute/ddp-study-server/assets/111771148/2e5fe423-7bee-4688-ae5f-da40ce30286f">